en/docs/deploy/deploy-a-containerized-choreo-component.md

```diff
@@ -46,9 +46,19 @@ Let's add a containerized REST API component by following these steps:
 
 6. Click **Authorize with GitHub** to allow the REST API to access your GitHub account.
 
+    If you have not already authorized Choreo applications, click **Authorize Choreo Apps** when prompted. 
     Once you perform the authorization, the **GitHub Account** field in the **Connect Repository** dialog box displays the GitHub account you authorized the REST API to access. 
 
-8. Select the following values to connect the repository.
+
+7. Select the following values to connect the repository:
+
+    !!! info
+        To successfully build your container with Choreo, it is essential that you explicitly define a User ID (`UID`) under the `USER` instruction in your Dockerfile. You can refer to the [sample Dockerfile](https://github.com/wso2/choreo-sample-apps/blob/main/go/rest-api/Dockerfile) for guidance.
+
+        To ensure that the defined USER instruction is valid, it must conform to the following conditions:
+            
+        - A valid User ID is a numeric value between 10000-20000, such as `10001` or `10500`.
+        - Usernames are not considered valid and should not be used, for example, `my-custom-user-12221` or `my-custom-user`.
```

Comment:
Should we say:
"The User ID should not include text values. For example, `my-custom-user-12221` or `my-custom-user`.